### PR TITLE
Update documentation for adding MarkAsJunk and MarkAsNotJunk APIs

### PR DESCRIPTION
**Pull request by kagunase**

### CODE REVIEW

File: api-reference/beta/api/message-markasnotjunk.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+## Response
+
+If successful, this method returns a `202 Accepted` response code and a [message](../resources/message.md) object in the response body.
+
+## Example
+
+### Request
+
+The following request moves the specified message to the Junk email folder, identified by its well-known folder name `inbox` and removes the sender from the list of blocked senders.
```
Comment (Jarbas-MSFT) [inline api-reference/beta/api/message-markasnotjunk.md:62]: The description isn't accurate. Please revise and update it.

File: api-reference/beta/api/message-markasjunk.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+  "sampleKeys": ["AAMkADhAAATs28OAAA="],
+  "name": "message_markasjunk"
+}-->
+
+```http
+POST https://graph.microsoft.com/beta/me/messages/AAMkADhAAATs28OAAA=/markAsJunk
+Content-type: application/json
+
+{
+  "moveToJunk": "true"
```
Comment (Jarbas-MSFT) [inline api-reference/beta/api/message-markasjunk.md:75]: Generally, boolean values are not passed as strings. Please, revise and update this JSON object.

Same for api-reference/beta/api/message-markasnotjunk.md

File: api-reference/beta/api/message-markasjunk.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+| Authorization | `Bearer {token}`. Required. |
+| Content-Type | `application/json`. Required. |
+
+## Request body
+
+In the request body, provide a JSON object with the following parameters.
+
+| Parameter   | Type |Description|
+|:---------------|:--------|:----------|
+|moveToJunk|Boolean|`True` moves the message to the **Junk Email** folder.|
```
Comment (Jarbas-MSFT) [inline api-reference/beta/api/message-markasjunk.md:52]: For consistency, I updated the appearances of moveToJunk and moveToInbox to be in camelCase format.

File: api-reference/beta/api/message-markasjunk.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
+---
```
Comment (Jarbas-MSFT) [inline api-reference/beta/api/message-markasjunk.md:1]: Please, include an update to the TOC.yml file in the BETA folder to surface these APIs (markAsJunk and markAsNotJunk) in the TOC.

**Comment by Jarbas-MSFT** [comment]
Hi @Lauragra, this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**